wifiphisher/pywifiphisher.py

```diff
@@ -523,7 +523,7 @@ def start(self):
         except (interfaces.InvalidInterfaceError,
                 interfaces.InterfaceCantBeFoundError,
                 interfaces.InterfaceManagedByNetworkManagerError) as err:
-            logger.exception("Setting {} to monitor mode".format(mon_iface))
+            logger.exception("Failed to set card to monitor mode")
             print ("[{0}!{1}] {2}").format(R, W, err)
```

Comment:
The logging message here is a little misleading since there are three type of exceptions here (i.e. `InvalidInterfaceError`, `InterfaceCantBeFoundError`, `InterfaceManagedByNetworkManagerError`), and the cause of exception not just because setting `monitor mode` fail. We can just log the below `print message` ?

Reply:
I do agree with you.